Fletcher32.cpp

```diff
@@ -52,6 +63,11 @@ void Fletcher32::add(const uint16_t * array, uint16_t length)
   }
 }
 
+uint32_t Fletcher32::getFletcher() {
+  _s1 = _s1 % 65535UL;
+  _s2 = _s2 % 65535UL;
+  return (((uint32_t)_s2) << 16) | _s1;
+};
```

Comment:
Optimized in PR14

Reply:
Agree. I will merge your PR into this one.